packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -1904,3 +1904,102 @@ let rows = [
   </TableBody>
 </TableView>
 ```
+
+## Expandable Rows <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+By default, TableView doesn't allow expandable rows, but this can be enabled using the `hasExpandableRows` prop.
+Use `defaultExpandedKeys` to provide a default set of expanded rows. Note that the value of the expanded keys must match the `key` prop of the Row.
```

Comment:
Would it be valuable to highlight that "all" will expand all the rows? What about highlighting that a expanded child row will only be visible if its parent is also in the expandedKeys set?